inst/markdown/compareScenarios2/cs2_10_ces_function.Rmd

```diff
@@ -0,0 +1,181 @@
+showLinePlots(data, "Internal|CES Function|CES Price|enb")
+
+```
+
+
+### Industry
+
+#### Total Industry
+
+```{r CES Prices Total Industry}
+showLinePlots(data, "Internal|CES Function|CES Price|ue_industry")
+```
+
+
+#### Steel
+
+```{r CES Prices Steel}
+
+showLinePlots(data, "Internal|CES Function|CES Price|feso_steel")
+showLinePlots(data, "Internal|CES Function|CES Price|feli_steel")
+showLinePlots(data, "Internal|CES Function|CES Price|fega_steel")
+showLinePlots(data, "Internal|CES Function|CES Price|feh2_steel")
+showLinePlots(data, "Internal|CES Function|CES Price|feel_steel_primary")
+showLinePlots(data, "Internal|CES Function|CES Price|feel_steel_secondary")
+
+
+showLinePlots(data, "Internal|CES Function|CES Price|en_steel_primary")
+
+showLinePlots(data, "Internal|CES Function|CES Price|ue_steel_primary")
+showLinePlots(data, "Internal|CES Function|CES Price|ue_steel_secondary")
+```
+
+#### Cement
+
+```{r CES Prices Cement}
+
+showLinePlots(data, "Internal|CES Function|CES Price|feso_cement")
+showLinePlots(data, "Internal|CES Function|CES Price|feli_cement")
+showLinePlots(data, "Internal|CES Function|CES Price|fega_cement")
+showLinePlots(data, "Internal|CES Function|CES Price|feh2_cement")
+showLinePlots(data, "Internal|CES Function|CES Price|feel_cement")
+
+showLinePlots(data, "Internal|CES Function|CES Price|en_cement")
+
+showLinePlots(data, "Internal|CES Function|CES Price|ue_cement")
+```
+
+#### Chemicals
+
+```{r CES Prices Chemicals}
+
+showLinePlots(data, "Internal|CES Function|CES Price|feso_chemicals")
+showLinePlots(data, "Internal|CES Function|CES Price|feli_chemicals")
+showLinePlots(data, "Internal|CES Function|CES Price|fega_chemicals")
+showLinePlots(data, "Internal|CES Function|CES Price|feh2_chemicals")
+showLinePlots(data, "Internal|CES Function|CES Price|feelhth_chemicals")
+
+showLinePlots(data, "Internal|CES Function|CES Price|en_chemicals")
+
+showLinePlots(data, "Internal|CES Function|CES Price|ue_chemicals")
+```
+
+
+#### Other Industry
+
+```{r CES Prices Other Industry}
+
+showLinePlots(data, "Internal|CES Function|CES Price|feso_otherInd")
+showLinePlots(data, "Internal|CES Function|CES Price|feli_otherInd")
+showLinePlots(data, "Internal|CES Function|CES Price|fega_otherInd")
+showLinePlots(data, "Internal|CES Function|CES Price|feh2_otherInd")
+showLinePlots(data, "Internal|CES Function|CES Price|feelhth_otherInd")
+
+showLinePlots(data, "Internal|CES Function|CES Price|en_otherInd")
+
+showLinePlots(data, "Internal|CES Function|CES Price|ue_otherInd")
+```
+
+
+## Marginal Rates of Substitution (selected)
+
+
+### Buildings
+
+```{r MRS Buildings}
+showLinePlots(data, "Internal|CES Function|MRS|feelhpb|fegab")
+showLinePlots(data, "Internal|CES Function|MRS|feelhpb|fehob")
+showLinePlots(data, "Internal|CES Function|MRS|feelhpb|fesob")
+showLinePlots(data, "Internal|CES Function|MRS|feelhpb|feelrhb")
+
+showLinePlots(data, "Internal|CES Function|MRS|feelh2b|fegab")
```

Comment:
`Internal|CES Function|MRS|feelh2b|fegab` does not exist. Do you mean `Internal|CES Function|MRS|feh2b|fegab`?

Reply:
thanks, solved it.